cyences_app_for_splunk/default/data/ui/views/cs_mssql.xml

```diff
@@ -62,6 +62,24 @@
         <latest>$tkn_time.latest$</latest>
       </search>
     </input>
+    <input type="multiselect" token="tkn_class_type" searchWhenChanged="true">
```

Comment:
Is this filter relevant to other database dashboard? 
If so, please add filter in that dashboard as well.